engine/class_modules/paladin/sc_paladin_retribution.cpp

```diff
@@ -932,19 +934,20 @@ void paladin_t::generate_action_prio_list_ret()
   // Items
 
   // special-cased items
-  std::unordered_set<std::string> special_items { "skulkers_wing", "macabre_sheet_music", "memory_of_past_sins", "dreadfire_vessel", "darkmoon_deck_voracity", "overwhelming_power_crystal", "spare_meat_hook", "grim_codex", "inscrutable_quantum_device", "sinful_gladiators_badge_of_ferocity" };
+  std::unordered_set<std::string> special_items { "skulkers_wing", "macabre_sheet_music", "memory_of_past_sins", "dreadfire_vessel", "darkmoon_deck_voracity", "overwhelming_power_crystal", "spare_meat_hook", "grim_codex", "inscrutable_quantum_device", "sinful_gladiators_badge_of_ferocity", "sinful_aspirants_badge_of_ferocity" };
 
-  cds -> add_action( "use_item,name=skulkers_wing" );
-  cds -> add_action( "use_item,name=macabre_sheet_music,if=buff.avenging_wrath.up|buff.crusade.up&buff.crusade.stack>=10" );
-  cds -> add_action( "use_item,name=memory_of_past_sins" );
-  cds -> add_action( "use_item,name=dreadfire_vessel" );
-  cds -> add_action( "use_item,name=darkmoon_deck_voracity,if=buff.avenging_wrath.up|buff.crusade.up&buff.crusade.stack>=10" );
-  cds -> add_action( "use_item,name=overwhelming_power_crystal,if=buff.avenging_wrath.up|buff.crusade.up&buff.crusade.stack>=10" );
-  cds -> add_action( "use_item,name=spare_meat_hook" );
-  cds -> add_action( "use_item,name=grim_codex" );
   cds -> add_action( "use_item,name=inscrutable_quantum_device,if=buff.avenging_wrath.up|buff.crusade.up&buff.crusade.stack>=10|cooldown.avenging_wrath.remains>45|cooldown.crusade.remains>45" );
+  cds -> add_action( "use_item,name=overwhelming_power_crystal,if=buff.avenging_wrath.up|buff.crusade.up&buff.crusade.stack>=10" );
+  cds -> add_action( "use_item,name=darkmoon_deck_voracity,if=buff.avenging_wrath.up|buff.crusade.up&buff.crusade.stack>=10" );
+  cds -> add_action( "use_item,name=macabre_sheet_music,if=buff.avenging_wrath.up|buff.crusade.up&buff.crusade.stack>=10" );
+  cds -> add_action( "use_item,name=dreadfire_vessel,if=(!talent.execution_sentence.enabled|debuff.execution_sentence.up|cooldown.execution_sentence.remains>25)" );
```

Comment:
as you've mentioned, ES doesn't affect these; let's get rid of these ES-related changes

Reply:
Agreed, I'll remove them.